core/server/api/v2/members.js

```diff
@@ -0,0 +1,35 @@
+const membersServiceUser = require('../../services/members').api.memberUserObject;
```

Comment:
Not the best name I guess, might need an update along with other namings.

Reply:
Is it worth leaving the name the same? rather than make a new one? Search and replace will be easier that way. nabd tho

Reply:
Yeah good point, updated.